src/reducers/medication-reducers.js

```diff
@@ -3,6 +3,7 @@ import queryString from 'query-string';
 import * as types from '../actions/action-types';
 import rxnorm from '../assets/medication-list';
 import { getConditionCodingFromCode } from './helpers/services-filter';
+import compareVersions from 'compare-versions';
```

Comment:
Looks like the build is failing on this line due to the linter, wanting absolute imports listed before relative imports.  You can confirm by running `npm run lint` on the project. If you move this line up, we should be good.

Reply:
Thanks for the tip. Done.